src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: Roboto, Arial, sans-serif;`.